src/mastercore.h

```diff
@@ -130,6 +130,10 @@ enum FILETYPES {
 #define OMNI_PROPERTY_MSC   1
 #define OMNI_PROPERTY_TMSC  2
 
+#include <boost/multiprecision/cpp_dec_float.hpp>
```

Comment:
Ahh - stretching my memory here but going back to the MetaDEx work I was doing on the UI, MetaDEx UI components include mastercore.h & I needed access to the math (specifically the precision) types we were using for MetaDEx trades so I could ensure the data I was displaying was accurate.  IIRC not needed for this build - strip them out you think?